ui-v2/app/templates/head.hbs

```diff
@@ -0,0 +1 @@
+<title>{{model.title}}</title>
```

Comment:
Was this file only needed for when you are using FastBoot? Maybe try and remove it and see if everything is fine. We don't run in FastBoot so if we don't need this file we could leave it out of the PR.

Reply:
I commented this file out and the page titles broke. This is a required file with or without FastBoot.